index.html

```diff
@@ -19,12 +19,18 @@
   <link rel="stylesheet" href="css/simple-grid.css">
   <title>Sunburst Smorgasbord | a tool to describe your relationships.</title>
   <meta name="description" content="A customizable smorgasbord to describe your relationships.">
+  <script src="https://cdn.rawgit.com/eligrey/canvas-toBlob.js/f1a01896135ab378aa5c0118eadd81da55e698d8/canvas-toBlob.js"></script>
+	<script src="https://cdn.rawgit.com/eligrey/FileSaver.js/e9d941381475b5df8b7d7691013401e171014e89/FileSaver.min.js"></script>
   <script src="https://d3js.org/d3.v7.min.js"></script>
   <script src="/js/sunburst.d3.js"></script>
     <script>
+
+      const mainWidth = 1152;
+      const mainHeight = 1152;
+
         d3.json("/assets/autoingredients.json")
         .then(function(flare) {
-            var chart = Sunburst(flare, {
+            let chart = Sunburst(flare, {
```

Comment:
You _could_ put all of this in a `main.js` file in the `js` folder if you want to move it out of `index.html` and then load it as the last script in the `<head>` tag